bottomless/src/replicator.rs

```diff
@@ -155,12 +155,19 @@ impl Options {
     pub fn from_env() -> Result<Self> {
         fn env_var(key: &str) -> Result<String> {
```

Comment:
I preemptively acked, but before we merge -- should all `env_var` callsites disallow empty vars?

Reply:
@avinassh can you double-check before I queue this?

Reply:
@psarna hey, let me double check all of the vars if we are using empty vars anywhere

Reply:
@psarna I have double checked all, we don't expect empty variables to be set anywhere in bottomless